the-9-exercises/src/components/Fetch.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import UserDisplayComponent from "./UserDisplayComponent";
+
+export default function Fetch() {
+  const [user, setUser] = useState([]);
+  const [random, setRandom] = useState(0);
+
+  useEffect(
+    function () {
+      fetch("https://random-data-api.com/api/users/random_user?size=10")
+        .then((res) => res.json())
+        .then((data) => setUser(data));
+    },
+    [random]
+  );
+
+  const DATA = user.map((item) => (
+    <UserDisplayComponent
```

Comment:
Suggestion: I think it would make it easy for you if you just pass the object and then desctructure it in 'UserDisplayComponent'